src/Sylius/Behat/Page/Admin/Crud/IndexPage.php

```diff
@@ -55,6 +55,20 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
+    public function countResources()
```

Comment:
Isn't this method redundant with `countItems` [method](https://github.com/Arminek/Sylius/blob/new-admin-zone/src/Sylius/Behat/Page/Admin/Crud/IndexPage.php#L88)?